docs/en/modules/gpio.md

```diff
@@ -98,18 +98,18 @@ gpio.serout(1,1,{8,18},8) -- serial 30% pwm 38k, lasts 8 cycles
 
 ## gpio.trig()
 
-Establish a callback function to run on interrupt for a pin.
-
-There is currently no support for unregistering the callback.
+Establish or clear a callback function to run on interrupt for a pin.
 
 This function is not available if GPIO_INTERRUPT_ENABLE was undefined at compile time.
 
 #### Syntax
-`gpio.trig(pin, type [, function(level)])`
+`gpio.trig(pin, [type [, function(level)]})`
 
 #### Parameters
 - `pin` **1~12**, IO index, pin D0 does not support interrupt.
-- `type` "up", "down", "both", "low", "high", which represent rising edge, falling edge, both edge, low level, high level trig mode correspondingly.
+- `type` "up", "down", "both", "low", "high", which represent rising edge, falling edge, both edge, 
+low level, high level trig mode correspondingly. If the type is "none" or omitted then the
+callback function is removed and the interrupt disabled.
 - `function(level)` callback function when triggered. The gpio level is the param. Use previous callback function if undefined here.
```

Comment:
From the documentation, the following ought to work:

gpio.trig(1, "both", function() print("edge") end)
gpio.trig(1, "up")

If the function isn't specified, then it persists from the previous call (according to the docs). The code will (I think) throw an error.